client/components/Model.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Model = (props) => {
+  const model = props.product;
+  console.log(model);
+  return (
+    <div>
+      <figure className="image is-64x64">
+        <img src={model.image} alt={model.model} />
```

Comment:
It sounds like model might not be the best name for this variable since a model is a property of the object being stored in it - is there something more meaningful we can call this? Or would there perhaps be some cleaner way of getting this information that circumvents storing it in a universal object?

Reply:
I see... There seems to be an overuse of this name, since our components are also 'models'. Perhaps auto would be a good replacement.